src/main/java/com/lauriewired/context/GhidraContext.java

```diff
@@ -51,4 +53,8 @@ default boolean isGuiMode() {
     default Optional<DataTypeManagerService> getDataTypeManagerService() {
         return Optional.empty(); // Not available in headless mode
     }
+
+    default Optional<PluginTool> getTool() {
+        return Optional.empty();
```

Comment:
It seems like this means runScript won't work on headless mode?  That's not good